proto/connector/v1/connector.proto

```diff
@@ -5,10 +5,10 @@ package connector.v1;
 import "google/protobuf/descriptor.proto";
 import "opencdc/v1/opencdc.proto";
 
-option (metadata_conduit_source_plugin_name) = "conduit.source.plugin.name";
-option (metadata_conduit_source_plugin_version) = "conduit.source.plugin.version";
 option (metadata_conduit_destination_plugin_name) = "conduit.destination.plugin.name";
 option (metadata_conduit_destination_plugin_version) = "conduit.destination.plugin.version";
+option (metadata_conduit_source_plugin_name) = "conduit.source.plugin.name";
+option (metadata_conduit_source_plugin_version) = "conduit.source.plugin.version";
```

Comment:
I used `buf format` to format our proto files so apparently it sorts options by alphabetical order.